pkg/transform/transform.go

```diff
@@ -0,0 +1,48 @@
+package transform
+
+import (
+	pb "github.com/expect-digital/translate/api/translate/translate/v1/gen"
+	"github.com/expect-digital/translate/pkg/model"
+)
+
+func MessageFromProto(m *pb.Message) model.Message {
+	return model.Message{
+		ID:      m.Id,
+		Message: m.Message,
+		Fuzzy:   m.Fuzzy,
+	}
+}
+
+func MessagesFromProto(m *pb.Messages) model.Messages {
+	var messagesToAdd []model.Message
+
+	for i := range m.Messages {
+		messagesToAdd = append(messagesToAdd, MessageFromProto(m.Messages[i]))
```

Comment:
You are appending index, not the message itself?
`for _,msg := range m.Messages {append...}`
Or maybe better
`for i,msg := range m.Messages {messagesToAdd[i]=msg}`

Reply:
I tested these functions locally and they work fine :)